### ISSUE TITLE
Deterministic behavior in the eval mode

### ISSUE THREAD

**Issue by KhrulkovV** [oskopek/mvae#4]
Hello!

Great code, thank you!

However, I have a small question: usually, when VAE is in the eval mode we only generate samples using the mean (so we don't sample from the latent distribution and simply forward `encoder(x)` to decoder). Is it possible to enforce such a behavior in your models?

Thanks in advance!

**Comment by oskopek** [comment]
Hi!

Sorry for the late reply. It should be possible, just change:

https://github.com/oskopek/mvae/blob/96d851ce446d8f7e975d82ec74caabb67cfbcd77/mt/mvae/models/vae.py#L75 line 75 and 76 to something like:
```
if self.training:
    z, data = q_z.rsample_with_parts()
    reparametrized.append(Reparametrized(q_z, p_z, z, data))
else:
    reparametrized.append(Reparametrized(q_z, p_z, q_z.mean, None))
```

Hope that works!
